xtask/src/release.rs

```diff
@@ -153,9 +201,47 @@ impl ReleaseTask {
         Ok(remote)
     }
 
-    /// Get the tag name for this release.
-    fn tag_name(&self) -> String {
-        format!("{}-{}", self.local_crate.name, self.local_crate.version)
+    /// Commit and push all the changes in the git repository.
+    fn commit(&self) -> Result<()> {
+        let mut input = String::new();
+        let stdin = stdin();
+
+        print!("Ready to commit the changes. [continue/abort/DIFF]: ");
+        stdout().flush()?;
+
+        let mut handle = stdin.lock();
+        handle.read_line(&mut input)?;
+
+        match input.trim().to_ascii_lowercase().as_str() {
+            "c" | "continue" => {}
+            "a" | "abort" => {
+                return Err("User aborted commit".into());
+            }
+            _ => {
```

Comment:
I find it a bit weird that random input triggers a diff here. Ideally there wouldn't be a default and the question would be repeated if none of `["c", "continue", "a", "abort", "d", "diff"]` is given. A default of abort would be fine too.